DatadogRUM/Sources/Instrumentation/Resources/URLSessionRUMResourcesHandler.swift

```diff
@@ -114,6 +114,23 @@ internal final class URLSessionRUMResourcesHandler: DatadogURLSessionHandler, RU
             interception.completion?.error
         ) ?? [:]
 
+        var originalRequest = interception.request.unsafeOriginal
+
+        // Extract GraphQL attributes from request headers
+        var combinedAttributes = userAttributes
+        if let operationName = originalRequest.value(forHTTPHeaderField: "_dd-custom-header-graph-ql-operation-name") {
+            combinedAttributes[CrossPlatformAttributes.graphqlOperationName] = operationName
+        }
+        if let operationType = originalRequest.value(forHTTPHeaderField: "_dd-custom-header-graph-ql-operation_type") {
```

Comment:
This one uses `_` while others use `-`; is it expected?
```suggestion
        if let operationType = originalRequest.value(forHTTPHeaderField: "_dd-custom-header-graph-ql-operation-type") {
```